releases/0.3.0.heads.md

```diff
@@ -0,0 +1,134 @@
+* Improve project structure introducing subdirectory per architecture.
+* Replace some hard-coded paths with variables.
+
+### Upstream status
+
+#### Pending
+
+Lines added: 14843
+Lines removed: 25
+
+#### Active
+
+* [55038: src/lib/fmap.c: use le*toh() functions where needed](https://review.coreboot.org/c/coreboot/+/55038)
+* [55039: util/cbfstool/flashmap/fmap.c: fix fmaptool endianness bugs on BE](https://review.coreboot.org/c/coreboot/+/55039)
+
+Lines added: 49
+Lines removed: 38
+
+#### Merged
+
+* [55037: ppc64/byteorder.h: define use of big endian](https://review.coreboot.org/c/coreboot/+/55037)
+* [55040: commonlib/lz4_wrapper.c: do not use bitfields](https://review.coreboot.org/c/coreboot/+/55040)
+* [55041: docs/flashmap: state the endianness of FMAP](https://review.coreboot.org/c/coreboot/+/55041/3)
+
+Lines added: 36
+Lines removed: 41
+
+### Download firmware images
+
+#### Release binaries
+
+* [dasharo-trustworthy-computing-v0.3.0.rom]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.SHA256]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.SHA256.sig]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.signed.ecc]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.signed.ecc.SHA256]()
+* [dasharo-trustworthy-computing-v0.3.0.rom.signed.ecc.SHA256.sig]()
+
+See how to verify signatures on
+
+[![asciicast](https://asciinema.org/a/XTwZHG3RMWe0Mm2VrV7VPloYB.svg)](https://asciinema.org/a/XTwZHG3RMWe0Mm2VrV7VPloYB)
+
+### Release statistics
+
+In this release release: \
+<span style="color:yellow">153</span> files were changed \
+<span style="color:lightgreen">3769</span> lines of code were added \
+<span style="color:orangered">408</span> lines of code were removed
+Check the statistics with:
+
+```
+git diff --stat 2918bcd dasharo-trustworthy-computing-v0.3.0
+```
+
+### Hardware configuration
+
+Configuration with a single IBM POWER9 64bit CPU is supported.\
+Dual CPU setup not supported currently.
+
+Following RAM configurations were tested and are proved to be properly initialized.
+<pre>
+MCS0, MCA0
+   DIMM0: <a href=https://www.samsung.com/semiconductor/dram/module/M393A2K40CB2-CTD>1Rx4 16GB PC4-2666V-RC2-12-PA0</a>
+   DIMM1: not installed
+MCS0, MCA1
+   DIMM0: <a href=https://www.crucial.com/memory/server-ddr4/mta9asf1g72pz-2g6j1>1Rx8 8GB PC4-2666V-RD1-12</a>
+   DIMM1: not installed
+MCS1, MCA0
+   DIMM0: <a href=https://www.samsung.com/semiconductor/dram/module/M393A4K40CB2-CTD>2Rx4 32GB PC4-2666V-RB2-12-MA0</a>
+   DIMM1: not installed
+MCS1, MCA1
+   DIMM0: <a href=https://mis-prod-koce-homepage-cdn-01-blob-ep.azureedge.net/web/static_file/12701730956286135.pdf>2Rx8 16GB PC4-2666V-RE2-12</a>
+   DIMM1: not installed
+</pre>
+
+All 3 major DRAM vendors are supported, namely Samsung, Micron and Hynix.
+
+### Building binary using source code:
+
+In order to build Heads, use the following command:
+
+```
+make BOARD=talos-2_server
+```
+#### Running the Heads on Talos II
+pflash -E -p /tmp/flash.pnor
```

Comment:
@IgorBagnucki Is this intentional??

Reply:
@macpijan It is compliant with @SergiiDmytruk  [instructions](https://jira-3mdeb.atlassian.net/browse/IOP-153?focusedCommentId=64204)
Not sure if we can run Heads with current state of coreboot support though.

Reply:
@IgorBagnucki I'm complaining for style, not the content. Why is this on top, directly after the header, with no code block and description? Isn't it essentialy the same stuff as in the 3rd point few lines below?

Reply:
I get it. It was not intentional to leave this in the doc